.eslintrc.json

```diff
@@ -9,6 +9,15 @@
   "rules": {
     "prefer-const": "error",
     "indent": ["error", 2],
+    "@stylistic/js/max-len": [
+      "error",
+      {
+        "code": 80,
+        "tabWidth": 2,
+        "ignoreComments": true,
+        "ignorePattern": "^import\\s.+\\sfrom\\s.+;$"
+      }
+    ],
```

Comment:
This allows to keep `max-len` but disable locations where it conflicts with prettier auto-format